config/locales/translations/zendesk_apps_support.yml

```diff
@@ -131,12 +131,20 @@ parts:
       screenshot: "https://drive.google.com/open?id=1iEx7f7PYnum_qDQESD3Gz2PwLDza3zI0"
   - translation:
       key: "txt.apps.admin.error.app_build.banner.invalid_format"
-      title: "App builder job: Banner image invalid format error"
+      title: "App builder job: banner image invalid format error"
       value: "Banner image must be a PNG file."
   - translation:
       key: "txt.apps.admin.error.app_build.banner.invalid_size"
-      title: "App builder job: Banner image invalid size error"
+      title: "App builder job: banner image invalid size error"
       value: "Invalid banner image dimensions. Must be %{required_banner_width}x%{required_banner_height}px."
+  - translation:
+      key: "txt.apps.admin.error.app_build.screenshots.excessive_files"
+      title: "App builder job: package contains too many screenshots"
```

Comment:
Can you upload screenshot please?
Does the `%{file_count}` replaced with a number?
Also what will be inserted into `%{supported_count}`?